config/jobs/kubernetes-csi/gen-jobs.sh

```diff
@@ -131,22 +131,55 @@ resources_for_kubernetes () {
     case $kubernetes in master|$experimental_k8s_version|release-*) cat <<EOF
 resources:
   requests:
-    # these are both a bit below peak usage during build
-    # this is mostly for building kubernetes
-    memory: "9000Mi"
-    # during the tests more like 3-20m is used
-    cpu: 2000m
+    memory: "12Gi"
+    cpu: 6
+  limits:
+    memory: "12Gi"
+    cpu: 6
 EOF
                             ;;
                             *) cat <<EOF
 resources:
   requests:
-    cpu: 2000m
+    memory: "9Gi"
+    cpu: 4
+  limits:
+    memory: "9Gi"
+    cpu: 4
```

Comment:
These resources seem too much. The suggestion from https://github.com/kubernetes/test-infra/issues/29722#issuecomment-1593055040 is to start with 2 CPUs (which mirrors the previous 2000m) and 4GB, then watch https://monitoring-eks.prow.k8s.io/d/96Q8oOOZk/builds?from=now-24h&to=now to see the actual usage.

Reply:
The original requests made for all csi jobs were 2 CPU and 9 GB. I agree that the 4 cpu and 6 cpu for build jobs is likely a bit high. I'll probably lower it to 2 and 4 CPU respectively.

Reply:
> The original requests made for all csi jobs were 2 CPU and 9 GB.

No, not quite. That was only for jobs which need to build Kubernetes from source. Everything else used "cpu: 2000m".

Let's use the original `2 cpu, 9 GiB` for build jobs and `2 cpu, 4 GiB` for the rest.

Reply:
Ok, thanks for clarifying! Will update.